drivers/overlay/ov_network.go

```diff
@@ -298,6 +300,20 @@ func (n *network) initSubnetSandbox(s *subnet) error {
 	return nil
 }
 
+func (n *network) cleanupStaleSandboxes() {
+	filepath.Walk(filepath.Dir(osl.GenerateKey("walk")),
+		func(path string, info os.FileInfo, err error) error {
+			_, fname := filepath.Split(osl.GenerateKey(fmt.Sprintf("%d-%s", 1, n.id)))
+			pattern := strings.Split(fname, "-")[1][:5]
+			if strings.Contains(info.Name(), pattern) {
```

Comment:
I think it would be better if we check for the full network ID contains the full filename (after the marker).
That will take care of also the insanely high epoch number.